src/content/content.npmjs.js

```diff
@@ -4,18 +4,20 @@ import { mountContentScript, reloadWhenURLChanged } from './content';
 import { fetchPackageInfo } from './content-events';
 import { urlParsers } from './registry/npm';
 
+const CSS_SELECTOR_COLLABORATORS = '#collaborators';
+
 const addPackageReport = async (packageID) => {
   // remove an old package report (if exists)
   const currPackageReport = document.getElementsByTagName(packageReportTagName);
   if (currPackageReport?.length) {
     currPackageReport.item(0).remove();
   }
 
-  const repository = await waitForElement('#repository', document.querySelector('#main'));
+  const collaborators = await waitForElement(CSS_SELECTOR_COLLABORATORS, document.querySelector('#main'));
   const packageReport = document.createElement(packageReportTagName);
   packageReport.setAttribute('package-type', packageID.type);
   packageReport.setAttribute('package-name', packageID.name);
-  repository.parentElement.insertBefore(packageReport, repository);
+  collaborators.parentElement.parentElement.prepend(packageReport);
 };
```

Comment:
did you change the location of Overlay?

Is the "collaborators" part always there?

Reply:
yes always there "collaborators". 
even if have only one collaborator.
I believe it impossible to have zero collaborators. 
not see a page like this. and it means no one uploads the project.

the location it same.
![Untitled1](https://github.com/os-scar/overlay/assets/44925669/90d17cb8-e48e-46ea-a47b-cfb95f05cafb)g

Reply:
I prefer it to be under the `install` and its command.

What do you think about `insertAfter` the `<p>` of `collaborators.parentElement.parentElement` (the `<p>` is the install command)?

Reply:
accepted